src/main/java/application/AbstractBaseController.java

```diff
@@ -157,6 +161,10 @@ private String getExceptionEmailBody(Exception exception){
         return "<html>" +
                 "<h3>Message</h3>" +
                 "<p>" + exception.getMessage() + "</p>" +
+                "<h3>Request URI</h3>" +
+                "<p>" + req.getRequestURI() + "<p>" +
```

Comment:
yeah i thought i had. i'll retry. need to revisit this anyways since gmail isn't parsing the html correctly